roles/apim-gateway/tasks/main.yml

```diff
@@ -70,6 +70,20 @@
         dest: "{{ item.dest }}"
       loop: "{{ config_files }}"
 
+    - name: Create miscellaneous file directory
```

Comment:
Is this necessary?

Reply:
yes.

Reply:
There might be cases where different miscellaneous files are copied to locations outside `misc_file_copy_location`. I'm thinking that we could remove this and have the user create the relevant directories and set their own copy destinations.